concept/type/attributetype.feature

```diff
@@ -161,6 +161,10 @@ Feature: Concept Attribute Type
     When session opens transaction of type: write
```

Comment:
While we're here I think we should delete lines 159 and 160, they seem really superfluous (since they are duplicated just below)
```gherkin
    When session opens transaction of type: write
    Then attribute(email) as(string) put: alice@email.com; throws exception
```